tests/test_config.py

```diff
@@ -82,8 +83,8 @@ def test_config_defaults_set() -> None:
                     ],
                 },
             },
-            "async": {"concurrency": 10, "timeout": None},
-            "threading": {"max_workers": None},
+            "async": {"concurrency": 1 if IS_WASM else 10, "timeout": None},
+            "threading": {"max_workers": 1 if IS_WASM else None},
```

Comment:
I have the same question as above around what an appropriate `async.concurrency` value would be